code/controllers/subsystem/mapping.dm

```diff
@@ -844,6 +848,22 @@ GLOBAL_LIST_EMPTY(the_station_areas)
 
 			true_to_offset_planes[string_real] |= offset_plane
 
+/datum/controller/subsystem/mapping/proc/lazy_load_template(template_key)
```

Comment:
how appropriate is sleeping for like 10 seconds in the different places this is used?
loading post init will always take longer then loading pre init, because of changeturf issues

Reply:
I've noticed it's not actually taking very long to load the map. It's almost instant, locally atleast

Reply:
interesting